### DISCUSSION TITLE
Popolare un campo con unico valore

### DISCUSSION THREAD

**Discussion by pigreco** [ondata/guidaVisiData#52] [Domande] [answered]
Ho una tabella con 4 campi/attributi, il secondo non contiene valori e ho necessità di popolarlo con una stringa `QGIS 3.14 Pi`.

![image](https://user-images.githubusercontent.com/7631137/154686328-359d9a86-99b9-419f-a204-83322e50e05c.png)

Ho usato il `cerca e sostituisci` per ottenere il mio obiettivo, ovvero:
1. `gs` per selezionare tutto;
2. sulla campo da popolare: `g*` e
3. `transform column by regex:` ` /QGIS 3.14 Pi`

la magia funziona,
ma esiste un modo più diretto?

**Comment by aborruso** [comment, marked as answer]
Caro @pigreco , questa è modalità base:
- vai nella colonna di tuo interesse;
- selezioni tutto con `gs`;
- poi `g=`
- e poi inserisci la tua espressione python, che qui è semplicemente `"/QGIS 3.14 Pi"`

**Reply by pigreco** [reply]
Grazie mille!
